pkg/node/volumemgr.go

```diff
@@ -1383,6 +1388,51 @@ func (m *VolumeManager) checkWbtChangingEnable(ctx context.Context, vol *volumec
 	return true
 }
 
+func (m *VolumeManager) createFakeDeviceIfNecessary(log *logrus.Entry, vol *volumecrd.Volume) (string, error) {
+	fakeDevice, ok := vol.Annotations[fakeDeviceVolumeAnnotation]
+	fakeDeviceSrcFilePath := fakeDeviceSrcFileDir + vol.Name
+	var err error
+	if !ok {
+		fakeDevice, err = m.fsOps.CreateFakeDevice(fakeDeviceSrcFilePath)
```

Comment:
in the current code, if the file whose path is fakeDeviceSrcFilePath exists, we will directly use dd command to overwrite this file by /dev/zero and it can work normally. So it doesn't matter the existing file will block the function.